packages/hmr-plugin/src/utils/internals.ts

```diff
@@ -1,15 +1,15 @@
-import { HRM_RUNTIME } from '../symbols';
+import { HmrRuntime } from '../symbols';
 
 declare const window: any;
 
-export function hmrSetReloaded(value: boolean = true): void {
-  if (window[HRM_RUNTIME.STATUS]) {
-    window[HRM_RUNTIME.STATUS].hmrReloaded = value;
+export function setHmrReloadedTo(value = true): void {
```

Comment:
I prefer to have no default value here because the usage without the parameter does not make sense.
Also, what is the reason for having this as well as the `markApplicationAsHmrReloaded`?

Reply:
markApplicationAsHmrReloaded - initial default hmr meta information

Reply:
done